certbot/display/ops.py

```diff
@@ -152,7 +152,8 @@ def _filter_names(names):
 
     """
     code, names = z_util(interfaces.IDisplay).checklist(
-        "Which names would you like to activate HTTPS for?",
+        "Which names would you like to activate HTTPS for? "
+        "(Use spacebar to select/deselect options)",
```

Comment:
You should only add this extra text if the `IDisplay` is an instance of `display_util.NcursesDisplay`.

Something like this:
```
import 
msg = "Which names..."
disp = z_util(interfaces.IDisplay)
if isinstance(disp, display_util.NcursesDisplay)
    msg + = " (Use spacebar..."
disp.checklist(msg, ..)
```
To see why, try running certbot with `-t` :).